programs/server/Server.cpp

```diff
@@ -445,6 +455,39 @@ void checkForUsersNotInMainConfig(
     }
 }
 
+static void loadEncryptionKey(const std::string & key_command [[maybe_unused]], Poco::Logger * log)
+{
+#if USE_BASE64 && USE_SSL && USE_INTERNAL_SSL_LIBRARY
+
+    auto process = ShellCommand::execute(key_command);
+
+    std::string b64_key;
+    readStringUntilEOF(b64_key, process->out);
+    process->wait();
+
+    // turbob64 doesn't like whitespace characters in input. Strip
+    // them before decoding.
+    std::erase_if(b64_key, [](char c)
```

Comment:
It has an inconsistency: in this place, the encryption key is in base64 while for encrypted disks it is in hex or binary.

Reply:
And I think, it's better to read it in hex.